### PR TITLE
157148005 header

### PR DESCRIPTION
**Pull request by CodingCarlos**

### CODE REVIEW

**Review by DevStarlight — APPROVED** [review]

**Comment by CodingCarlos** [comment]
Test deployed in http://35.190.182.217:8080/

File: gulpfile.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ gulp.task('make-config', function () {
 });
 
 gulp.task('flavour', function () {
-	var flavour = process.env.FLAVOUR || 'vimojo';
+	var flavour = 'm4n';// process.env.FLAVOUR || 'vimojo';
```
Comment (DevStarlight) [inline gulpfile.js]: hmmm shouldn't this be deleted?

File: components/header-box/header-box.view.scss [hunk 2]
```diff
@@ -1,7 +1,10 @@
 .header-box-wrapper {
+	@extend %align-items-center;
+	background-color: $mainColor;
+	display: flex;
```
Comment (DevStarlight) [inline components/header-box/header-box.view.scss]: don't forget about display flex css compatibility

File: components/header-box/header-box.view.scss [hunk 3]
```diff
@@ -15,10 +18,11 @@
 		@extend %flex-direction-column;
 	    @extend %align-items-center;
 	    @extend %justify-content-center;
+	    align-self: flex-start;
```
Comment (DevStarlight) [inline components/header-box/header-box.view.scss]: don't forget about display flex css compatibility

File: flavour/m4n/components/header-box/header-box.view.scss [hunk 4]
```diff
@@ -1,7 +1,10 @@
 .header-box-wrapper {
+	@extend %align-items-center;
+	background-color: $mainColor;
+	display: flex;
```
Comment (DevStarlight) [inline flavour/m4n/components/header-box/header-box.view.scss]: don't forget about display flex css compatibility

File: flavour/m4n/components/header-box/header-box.view.scss [hunk 5]
```diff
@@ -15,10 +18,11 @@
 		@extend %flex-direction-column;
 	    @extend %align-items-center;
 	    @extend %justify-content-center;
+	    align-self: flex-start;
```
Comment (DevStarlight) [inline flavour/m4n/components/header-box/header-box.view.scss]: don't forget about display flex css compatibility

File: flavour/m4n/components/header-box/header-box.view.scss [hunk 6]
```diff
@@ -20,6 +20,7 @@
 	    @extend %justify-content-center;
 	    align-self: flex-start;
```
Comment (DevStarlight) [inline flavour/m4n/components/header-box/header-box.view.scss]: don't forget about display flex css compatibility

File: flavour/m4n/components/header-box/header-box.view.scss [hunk 7]
```diff
@@ -20,6 +20,7 @@
...
 		color: $backgroundLight;
+		display: flex;
```
Comment (DevStarlight) [inline flavour/m4n/components/header-box/header-box.view.scss]: don't forget about display flex css compatibility

File: components/header-box/header-box.view.html [hunk 8]
```diff
@@ -1,5 +1,5 @@
 <div class="header-box-wrapper">
-	<video no-controls autoplay src="/img/brand/header.mp4"></video>
+	<video no-controls autoplay="autoplay" src="/img/brand/header.mp4"></video>
```
Comment (DevStarlight) [inline components/header-box/header-box.view.html]: there's a bug in google chrome for what if video is not muted, chrome stops it.

**Pull request merged**